### PR TITLE
[Polymer UI] Add support for dark theme

### PR DESCRIPTION
**Pull request by yebrahim**
Most of this change is styling to use CSS3 variables instead of hard coded colors, then there's a `index.light.css` and a `index.dark.css` that specify values for these colors, then there's a little bit of logic on the server side to return the right file.

The theme can currently be switched manually by making a `POST` request to the settings API. Next I'll add the UI setting.

image

### CODE REVIEW

File: sources/web/datalab/polymer/index.dark.css [hunk 1]
```diff
@@ -12,16 +12,13 @@
...
-  right: 0;
-  min-width: 470px;
+:root {
+  --primary-accent-color: #222;
+  --primary-bg-color: #222;
+  --secondary-bg-color: #2b2b2b;
+  --primary-fg-color: #aaa;
+  --selection-bg-color: #444;
+  --selection-fg-color: #d68600;
+  --disabled-bg-color: var(--primary-bg-color);
```
Comment (jimmc) [inline sources/web/datalab/polymer/index.dark.css]: Why is there a disabled background color if it is always set to the same as the primary background color? If the expectation is that it is used with a different background color, such as secondary-bg-color, perhaps the name should be disabled-secondary-bg-color.

File: sources/web/datalab/polymer/index.dark.css [hunk 2]
```diff
@@ -12,16 +12,13 @@
...
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  min-width: 470px;
+:root {
+  --primary-accent-color: #222;
+  --primary-bg-color: #222;
+  --secondary-bg-color: #2b2b2b;
+  --primary-fg-color: #aaa;
```
Comment (jimmc) [inline sources/web/datalab/polymer/index.dark.css]: The order seems a bit random here. Maybe you could put all the primary colors together, or put all the backgrounds together, or use alpha order.

File: sources/web/datalab/polymer/index.dark.css (hunk 1 above)
Comment (yebrahim) [inline sources/web/datalab/polymer/index.dark.css]: It's just that it's a different state, so I think it's good to separate its variable out, even if it has the same value.
I'm ok removing this if you still don't think it should be there.

File: sources/web/datalab/polymer/index.dark.css (hunk 2 above)
Comment (yebrahim) [inline sources/web/datalab/polymer/index.dark.css]: Sure, alpha seems fine.

File: sources/web/datalab/polymer/index.dark.css (hunk 1 above)
Comment (jimmc) [inline sources/web/datalab/polymer/index.dark.css]: Not high priority for me, so it's OK as is.

**Pull request merged**